### ISSUE TITLE
ensure all crypto is forward secure

### ISSUE THREAD

**Issue by ioerror** [mumble-voip/mumble-iphoneos#86]
The crypto in mumble may or may not be forward secure. I think it should always be forward secure and there should be an obvious way to visually confirm it.

**Comment by ioerror** [comment]
( At the moment, I see the 'certificate' button but it doesn't tell me how the certificiate is used. )

**Comment by mkrautz** [comment]
Yes, the iOS client is quite lacking in server information as compared to the desktop version. The certificate view controller could also use some more details.

I hope to address it in the future...
